pkgs/by-name/bo/bombsquad/package.nix

```diff
@@ -0,0 +1,100 @@
+{
+  stdenv,
+  lib,
+  python3,
+  SDL2,
+  mesa,
+  libGL,
+  openal,
+  libvorbis,
+  fetchurl,
+  makeWrapper,
+  patchelf,
+  copyDesktopItems,
+  makeDesktopItem
+}:
+let
+  runtimeDependencies = [
+    python3
+    SDL2
+    libvorbis
+    mesa
+    stdenv.cc.cc.lib
+    libGL
+    openal
+  ];
+in
+stdenv.mkDerivation (finalAttrs: {
+  version = "1.7.33";
+  pname = "bombsquad";
+
+  src = fetchurl {
+    url = "http://files.ballistica.net/bombsquad/builds/BombSquad_Linux_x86_64_${finalAttrs.version}.tar.gz";
+    hash = "sha256-pfimBt1s4QP+6buPvTkSdmt5zj2NpXqJUfbxRirfz/o=";
+  };
+
+  sourceRoot = "BombSquad_Linux_x86_64_${finalAttrs.version}";
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  nativeBuildInputs = [
+    patchelf
+    makeWrapper
+    copyDesktopItems
+  ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "Bombsquad";
+      exec = "bombsquad";
+      icon = fetchurl {
+        url = "https://www.froemling.net/wp-content/uploads/2018/02/icon_clipped-1-150x150.png";
+        hash = "sha256-x08rekyPzHe2FdQzo1EIZ3Rgg/OAcTbWCHoiKuGRBvU=";
+      };
+      comment = finalAttrs.meta.description;
+      desktopName = "Bombsquad";
+      genericName = "Bombsquad";
+      startupWMClass = ".bombsquad-wrapped";
```

Comment:
Is this needed? This name is an implementation detail of `makeWrapper`, not being API-stable.

Reply:
It is needed so that the desktop environments recognize the application as opened. If you don't put a class, when you click the icon, desktop environments like gnome or KDE will show a different icon for the window instead of merging with the app icon. I just copy pasted the class that hyprland gave me in `hyprctl clients` command.

Reply:
https://askubuntu.com/questions/367396/what-does-the-startupwmclass-field-of-a-desktop-file-represent